mopidy/core/playback.py

```diff
@@ -251,6 +251,10 @@ def _on_about_to_finish(self):
         if self._state == PlaybackState.STOPPED:
             return
 
+        # Unless overridden by other calls (e.g. next / previous / stop) this
+        # will be the last position recorded until the track gets reassigned.
+        self._last_position = self._current_tl_track.track.length
```

Comment:
I'm putting a TODO to throw some sort of relevant exception whenever we get around fixing this.